reference-implementation/to-upstream-wpts/piping/error-propagation-backward.js

```diff
@@ -286,7 +286,7 @@ promise_test(t => {
   const ws = recordingWritableStream({
     write() {
       if (ws.events.length > 2) {
-        return delay(10).then(() => {
+        return flushAsyncEvents().then(() => {
```

Comment:
Okay, where we are not waiting for something to not happen, but instead just testing async behaviour, I have changed the flushAsyncEvents() to delay(0). delay(0) is currently the usual way to say "do this asynchronously" in these tests.

The net result of this is that a lot of the changes just end up changing delay(n) to delay(0), but that still satisfies the goal of reducing use of delay(0).